src/views/Home/index.js

```diff
@@ -41,13 +43,14 @@ export default class Home extends Component {
     const answer = await wellness.getStatus();
 
     if(answer.length > 0){
-      if (answer[0].answerValid === true) {
-        this.setState({ answered: true });
-        
-      }
-      if (answer[0].answerValid === false) {
-        this.setState({ answered: false });
-      }
+
```

Comment:
I think these 2 "if" statements could just be:

this.setState({ answered: answer[0].answerValid });